iogt/static/css/iogt.css

```diff
@@ -2583,3 +2583,7 @@ input[type=checkbox], input[type=radio] {
     margin-bottom: 12px
   }
 }
+
+.text-align-center{
```

Comment:
I suggest this be renamed to something that doesn't imply the stylistic choice that will be taken if there is no lead image. To keep our options open with regards to changing the look of IoGT (i.e. theming) we should just indicate that no lead image is available e.g. `.no-lead-image`.

Reply:
Okay got it, it's fixed now!